core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -200,7 +200,7 @@ private[spark] class TaskSchedulerImpl(
         val execId = taskIdToExecutorId(tid)
         backend.killTask(tid, execId, interruptThread)
       }
-      tsm.abort("Stage %s cancelled".format(stageId))
+      tsm.abort("Stage %d was cancelled".format(stageId))
```

Comment:
IMHO yes let's not bother with this sort of thing. It's a tiny change but doesn't improve anything and actually makes things a tiny bit more inconsistent.